gocd-server/tests/test.bats

```diff
@@ -0,0 +1,6 @@
+source "${BATS_TEST_DIRNAME}/../plan.sh"
+
+@test "Version matches" {
+  result="$(curl -s http://localhost:8153/go/admin/pipeline/new?group=defaultGroup| grep data-version-go-version | tr -d [:lower:],=,'"',-)"
+  [ "$result" = "${pkg_version}" ]
+}
```

Comment:
Tests should also include some process / service checks (as this is a service plan) as well as any confirming any ports are open. Since this curls an endpoint in the version check, it would be prudent to do the port open checks before version checking with curl.